setup.py

```diff
@@ -28,8 +29,8 @@
         'SrsOcGriddedHandler = aodndata.srs.srs_oc_gridded:SrsOcGriddedHandler',
         'SrsOcLjcoHandler = aodndata.srs.srs_oc_ljco:SrsOcLjcoHandler',
         'SrsOcSoopRadHandler = aodndata.srs.srs_oc_soop_rad:SrsOcSoopRadHandler',
-        'SstaarsHandler = aodndata.csiro.sstaars:SstaarsHandler',
-        'SoopCo2Handler = aodndata.soop.soop_co2:SoopCo2Handler'
```

Comment:
You removed SoopCo2Handler here, is this normal?

Reply:
no I moved it above to be ordered properly